esx_service/utils/auth.py

```diff
@@ -34,6 +34,7 @@
 
 SIZE = 'size'
 DEFAULT_TENANT = '_DEFAULT'
+DEFAULT_TENANT_UUID = '_DEFAULT_TENANT_UUID'
```

Comment:
Yes, this would create a folder "_DEFAULT_TENANT_UUID" and soft link _DEFAULT.
For the second question, I have no preference, either way looks good to me.